modules/transactions/src/main/java/org/apache/ignite/internal/tx/impl/TxManagerImpl.java

```diff
@@ -111,9 +109,7 @@ public boolean changeState(UUID txId, TxState before, TxState after) {
     @Override
     public CompletableFuture<Lock> writeLock(UUID lockId, ByteBuffer keyData, UUID txId) {
```

Comment:
It's out of the scope of you ticket, however let's just remove writeLock and readLock methods from TxManager. They are deprecated and never used.